pkg/integrations/install/install.go

```diff
@@ -33,6 +34,7 @@ import (
 	_ "github.com/grafana/agent/pkg/integrations/v2/agent" // register agent
 	_ "github.com/grafana/agent/pkg/integrations/v2/apache_http"
 	_ "github.com/grafana/agent/pkg/integrations/v2/app_agent_receiver" // register app_agent_receiver
+	_ "github.com/grafana/agent/pkg/integrations/v2/blackbox_exporter"
 	_ "github.com/grafana/agent/pkg/integrations/v2/eventhandler"
 	_ "github.com/grafana/agent/pkg/integrations/v2/snmp_exporter"
 	_ "github.com/grafana/agent/pkg/integrations/v2/vmware_exporter"
```

Comment:
Can we clean this up a little bit and make sure all of these have comments like the v1 integrations do?